web/src/toga_web/window.py

```diff
@@ -1,11 +1,14 @@
 from toga.command import Group, Separator
+from toga.constants import WindowState
 from toga.types import Position, Size
 from toga_web.libs import create_element, js
 
 from .screens import Screen as ScreenImpl
 
 
 class Window:
+    _PLATFORM_ALLOWS_CLOSE = False
```

Comment:
What is this for?

Reply:
Removed it. It's an artifact from the previous main_window closing bug repair.